notebooks/standardize_events_data.ipynb

```diff
@@ -70,15 +279,13 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
+   "execution_count": 5,
    "id": "03358fb0-dd7b-4c13-a57a-4703a2622ecd",
    "metadata": {},
    "outputs": [],
    "source": [
     "def decode_byte_string(value):\n",
-    "    if isinstance(value, bytes):\n",
-    "        return value.decode(\"utf-8\")\n",
-    "    return value"
+    "    return value.decode(\"utf-8\")  "
```

Comment:
Do you need a function for this? Isn't it simpler to use the `decode` method directly?